github-user-main/retrieve-user/retrieveUserHandler.ts

```diff
@@ -0,0 +1,15 @@
+import { IArgumentsRetrieve } from '../../@types/types'
+import queryUserData from '../../db/queryUserData'
+
+export const retrieveUserHandler = async (argv: IArgumentsRetrieve) => {
+	try {
+		const users = await queryUserData(argv)
+		console.table(users)
+	} catch (error) {
+		const errorMessage = error.isAxiosError ? error.response.data : error.message
+		console.error(errorMessage)
+		
```

Comment:
I'm seeing multiple exit points.
How does this affect each function re-usability?

Reply:
I took it off. 
It would make the function re-usability very hard cause each return would need to be expect from the caller function. Not good.